libr/core/anal_vt.c

```diff
@@ -36,8 +39,10 @@ static int inTextSection(RCore *core, ut64 curAddress) {
 }
 
 static int valueInTextSection(RCore *core, ut64 curAddress) {
+	int sz = R_DIM (8, 1, 8);
 	//value at the current address
-	ut64 curAddressValue = r_io_read_i (core->io, curAddress, 8);
+	ut64 curAddressValue;
+	r_io_read_at (core->io, curAddress, (ut8 *)&curAddressValue, sz);
```

Comment:
drop this change. r_io_read_i wasnt removed. i think we should drop all changes in rbin because are unrelated to siol, and what matters was the offset->paddr